suites/pacific/rados/tier-3_rados_test-pool-functionalities.yaml

```diff
@@ -266,3 +266,12 @@ tests:
         test_autoscaler_bulk_feature: true
         pool_name: test_bulk_features
         delete_pool: true
+  - test:
+      name: PG number maximum limit check
+      module: pool_tests.py
+      desc: Check the pg_num maximut limit is <=128
+      polarion-id: CEPH-83574909
```

Comment:
Test case still in draft state. Please update

Reply:
Test case status updated for the review.

Reply:
Test case approved!